scripts/baseDemo.ts

```diff
@@ -0,0 +1,363 @@
+    graph(id: $graphId) {
+      updateLinterConfiguration(changes: $changes) {
+        rules {
+          rule
+        }
+      }
+    }
+  }`);
+
+function createGraph(id: string) {
+  const uniqueId = v4().replace('-', '').substring(0, 6);
+  return client
+    .mutate({
+      mutation: CREATE_GRAPH,
+      variables: {
+        accountId: id,
+        newServiceId: `spotify-demo-graph-${uniqueId}`,
+        name: 'Spotify Demo Graph',
+      },
+    })
+    .then((r) => ({
+      newService: r.data?.newService,
+    }));
+}
+
+async function createOperationCollection(graphId: string) {
+  const { createOperationCollection } = await client
+    .mutate({
+      mutation: CREATE_OPERATION_COLLECTION,
+      variables: {
+        isSandbox: false,
+        isShared: true,
+        name: 'Example Operations',
+        variantRefs: `${graphId}@${graphVariant}`,
+      },
+    })
+    .then((r) => r.data);
+  return await client
+    .mutate({
+      mutation: ADD_OPERATION_TO_COLLECTION,
+      variables: {
+        operationCollectionId: createOperationCollection.id,
+        operations: [
+          {
+            name: 'MyPlaylists',
+            document: {
+              body: `query MyPlaylists($offset: Int, $limit: Int) {
+    me {
+      playlists(offset: $offset, limit: $limit) {
+        pageInfo {
+          offset
+          limit
+          hasNextPage
+        }
+        edges {
+          node {
+            id
+            name
+            uri
+          }
+        }
+      }
+    }
+  }`,
+            },
+          },
+          {
+            name: 'PlaybackState',
+            document: {
+              body: `subscription PlaybackState {
+    playbackStateChanged {
+      isPlaying
+      progressMs
+      item {
+        name
+      }
+    }
+  }
+  `,
+            },
+          },
+          {
+            name: 'PausePlayback',
+            document: {
+              body: `mutation PausePlayback {
+    pausePlayback {
+      playbackState {
+        isPlaying
+      }
+    }
+  }`,
+            },
+          },
+          {
+            name: 'ResumePlayback',
+            document: {
+              body: `mutation ResumePlayback {
+    resumePlayback {
+      playbackState {
+        isPlaying
+      }
+    }
+  }`,
+            },
+          },
+        ],
+      },
+    })
+    .then((r) => r.data);
+}
+
+function updateLinterConfig(graphId: string) {
+  return client.mutate({
+    mutation: UPDATE_LINTER_CONFIG,
+    variables: {
+      graphId,
+      changes: {
+        rules: [
+          {
+            rule: 'FIELD_NAMES_SHOULD_BE_CAMEL_CASE',
+            level: 'ERROR',
+          },
+        ],
+      },
+    },
+  });
+}
+
+function updateReadme(
+  graphId: string,
+  readme: string,
+  ops: { id: string; name: string }[]
+) {
+  ops.forEach(({ id, name }) => {
+    readme = readme.replace(`<replace-${name}>`, `{{ operation.${id} }}`);
```

Comment:
Not sure if these `replace-` are only used once in the doc or not, but you might consider switching this to `replaceAll` to ensure all instances of `replace-${name}` are replaced.